packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -770,6 +772,8 @@ function prepareFreshStack(root, expirationTime) {
     root.timeoutHandle = noTimeout;
     // $FlowFixMe Complains noTimeout is not a TimeoutID, despite the check above
     cancelTimeout(timeoutHandle);
+    // $FlowFixMe Complains noTimeout is not a TimeoutID, despite the check above
+    SuspenseTimeouts.delete(timeoutHandle);
```

Comment:
It would've been nice to only have this for when in a jest environment. This is a place the testing specific build would have been great.